package.json

```diff
@@ -91,6 +91,9 @@
     "react-router-dom": "^5.1.2",
     "rxjs": "^6.5.5"
   },
+  "peerDependencies": {
+    "@openmrs/esm-patient-chart-app": "0.0.1"
```

Comment:
Seems like an anti-pattern.

Right now `@openmrs/esm-patient-chart-app` depends on `@openmrs/esm-patient-chart-widgets`, but not the other way round.

Plus we are actually working towards that the former dependency is not there.

Reply:
@FlorianRappl,
Thanks for your inputs. we have removed it from `peerDependencies` as discussed.